### ISSUE TITLE
404 page has no data

### ISSUE THREAD

**Issue by LasseRosenow** [hamedasemi/lit-element-router#10]
My router looks like this:

```
router([{
    name: 'news',
    pattern: 'news'',
    data: { title: 'News' }
}, {
    name: 'notfound',
    pattern: '*',
    data: { title: 'Error 404!' }
}], (route, params, query, data) => {
    this.route = route
    console.log(data)
})
```

When I go to /news, everything works as expected and the console logs:
```
{
    title: "News"
}
```

But when ever I open any link that will show my 404 page like "/sdf", my console logs:
```
{}
```

**Comment by hamedasemi** [comment]
I can see this problem and will be fixed with next patch.

**Comment by hamedasemi** [comment]
Fixed and published under 1.2.5 version
